crowdkit/aggregation/classification/multi_binary.py

```diff
@@ -0,0 +1,110 @@
+     from the output of the basic aggregator used for its intended purpose, since each class generates a binary
+     classification task, and therefore it is considered separately. For example, some objects may not have labels.
+
+     {% endnote %}
+
+    Args:
+        aggregator: A type of aggregator class that will be used for each binary classification.
+
+        args: (optional) Dictionary of args to be passed in aggregators, if such needed.
+
+    Examples:
+        >>> import pandas as pd
+        >>> from crowdkit.aggregation import MultiBinary, MajorityVote
+        >>> df = pd.DataFrame(
+        >>>     [
+        >>>         ['t1', 'w1', ['house', 'tree']],
+        >>>         ['t1', 'w2', ['house']],
+        >>>         ['t1', 'w3', ['house', 'tree', 'grass']],
+        >>>         ['t2', 'w1', ['car']],
+        >>>         ['t2', 'w2', ['car', 'human']],
+        >>>         ['t2', 'w3', ['train']]
+        >>>     ]
+        >>> )
+        >>> df.columns = ['task', 'worker', 'label']
+        >>> result = MultiBinary(DawidSkene, {'n_iter': 10}).fit_predict(df)
+
+    Attributes:
+        labels_ (typing.Optional[pandas.core.series.Series]): Tasks' labels.
+            A pandas.Series indexed by `task` such that `labels.loc[task]`
+            is the tasks' aggregated labels.
+
+        aggregators_ (dict[str, BaseClassificationAggregator]): Labels' aggregators matched to classes.
+            A dictionary that matches aggregators to classes.
+            The key is the class found in the source data,
+            and the value is the aggregator used for this class.
+            The set of keys is all the classes that are in the input data.
+    """
+
+    args: tp.Dict[str, tp.Any] = attr.ib(validator=attr.validators.instance_of(dict), default={})
+    aggregators_: tp.Dict[str, BaseClassificationAggregator] = dict()
+    aggregator: type = attr.ib(default=MajorityVote)
+
+    def fit(self, data: pd.DataFrame) -> 'MultiBinary':
+        """Fit the aggregators.
+
+        Args:
+            data (DataFrame): Workers' labeling results.
+                A pandas.DataFrame containing `task`, `worker` and `label` columns.
+                'label' column should contain list of labels, e.g. ['tree', 'house', 'car']
+        """
+
+        data = data[['task', 'worker', 'label']]
+        mlb = MultiLabelBinarizer()
+        binarized_labels = mlb.fit_transform(data['label'])
+        task_to_labels: tp.DefaultDict[tp.Union[str, float], tp.List[tp.Union[str, float]]] = defaultdict(list)
+
+        for i, label in enumerate(mlb.classes_):
+            single_label_df = data[['task', 'worker']]
+            single_label_df['label'] = binarized_labels[:, i]
+
+            label_aggregator = self.aggregator(**self.args)
+            label_aggregator.fit_predict(single_label_df)
+            self.aggregators_[label] = label_aggregator
+            for task, label_value in dict(label_aggregator.labels_).items():
```

Comment:
Can we enumerate it more elegant than by building a `dict` and then iterating over `items()`?